foundation_cms/templatetags/app_environment_tags.py

```diff
@@ -0,0 +1,19 @@
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def environment_prefix(context):
+    env_prefix = ""
+    app_env = get_app_environment()
+    if app_env == "Staging":
+        env_prefix = "[S]"
+    elif app_env == "Review":
+        env_prefix = "[RA]"
+    return env_prefix
+
+
+def get_app_environment():
+    return settings.APP_ENVIRONMENT
```

Comment:
While the `environment_prefix` tag is not exactly needed for this work, it seemed like something useful to bring over from the old site for later work!